docs/contributing/code-of-conduct.md

```diff
@@ -1,76 +1,69 @@
 ---
-title: Gatsby Contributor Covenant Code of Conduct
+title: Kodeks postępowania współautora Gatsby 
 ---
 
-## Our Pledge
+## Nasze Ślubowanie
 
-In the interest of fostering an open and welcoming environment, we as
-contributors and maintainers pledge to making participation in our project and
-our community a harassment-free experience for everyone, regardless of age, body
-size, disability, ethnicity, gender identity and expression, level of
-experience, nationality, personal appearance, race, religion, or sexual identity
-and orientation.
+W interesie kreowania otwartego i przyjaznego środowiska my jako współautorzy 
```

Comment:
```suggestion
W interesie kreowania otwartego i przyjaznego środowiska my, jako współautorzy,
```

Reply:
jest to jakaś konkretna zasada interpunkcyjna? dałbyś radę może znaleźć gdzieś podobny przykład?

Reply:
Szukałem ale w sumie nie ma żadnej informacji na ten temat. Jakoś tak po prostu brzmiało to dla mnie logicznie z przecinkami, ale być możę się mylę, więc możemy to olać :)